v2/api/kubernetesconfiguration/v1api20230501/structure.txt

```diff
@@ -255,3 +255,420 @@ Extension_Spec_ARM: Object (5 properties)
         ├── "Key"
         ├── "ManagedIdentity"
         └── "User"
+FluxConfiguration: Resource
+├── Spec: Object (14 properties)
+│   ├── AzureBlob: *Object (9 properties)
+│   │   ├── AccountKey: *genruntime.SecretReference
+│   │   ├── ContainerName: *string
+│   │   ├── LocalAuthRef: *string
```

Comment:
Any idea what this is, and if it's a secret?

Reply:
It's the "Name of a local secret on the Kubernetes cluster to use as the authentication secret rather than the managed or user-provided configuration secrets."